packages/@tinacms/app/package.json

```diff
@@ -9,6 +9,7 @@
     "typescript": "^4.6.4"
   },
   "dependencies": {
+    "@codemirror/language": "6.0.0",
```

Comment:
was this meant to be added?

Reply:
Yeah it's used by the GraphiQL package, though they weren't declaring it properly